release/models/bgp/openconfig-bgp-common.yang

```diff
@@ -698,17 +704,52 @@ submodule openconfig-bgp-common {
     description
       "Parameters controlling usage of  of DMZ Link-Bandwidth
       extended community in pultipath RIB/FIB formation";
+
     leaf enabled {
       type boolean;
       description
-      "When set to TRUE, BGP multiplepath shall distributed traffic
-      load among contributing routes proportionally to value of
-      Local Administrator subfield of link-bandwidth extended
-      community [draft-ietf-idr-link-bandwidth-07].
-      This leaf has no effect if BGP multi-path is disabled or
-      if maximum-path attribute of BGP multi-path value is set
-      to 1";
+        "When set to TRUE, BGP multiplepath shall distributed traffic
+        load among contributing routes proportionally to value of
+        Local Administrator subfield of link-bandwidth extended
+        community [draft-ietf-idr-link-bandwidth-07].
+        This leaf has no effect if BGP multi-path is disabled or
+        if maximum-path attribute of BGP multi-path value is set
+        to 1";
+    }
+
+    leaf cumulative {
```

Comment:
I would recommend "send-cumulative" to represent the role in how this is transmitted.
Note that in junos, inconsistent configuration of this feature at the neighbor level can result in peergroup splits since it's "hidden policy".

Reply:
Done